lib/awful/titlebar.lua

```diff
@@ -101,6 +104,7 @@ local function new(c, args)
         c:connect_signal("focus", update_colors)
         c:connect_signal("unfocus", update_colors)
     else
+        args.context = {}
```

Comment:
Not the cleanest code on earth, but it is a placeholder until the "real" context is created.